UnityProject/Assets/Scripts/Items/Medical/Implanter.cs

```diff
@@ -0,0 +1,137 @@
+using UnityEngine;
+using HealthV2;
+using Items;
+
+public class Implanter : MonoBehaviour, ICheckedInteractable<HandApply>, ICheckedInteractable<InventoryApply>, ICheckedInteractable<HandActivate>
+{
+	[SerializeField]
+	private GameObject implantObject = null;
+
+	[SerializeField]
+	private int timeToImplant = 5;
+
+	[SerializeField]
+	private ItemTrait ImplantableTrait = null;
+
+	[SerializeField]
+	private SpriteHandler spriteHandler;
+
+	private ItemStorage itemStorage;
+	private bool primed = false;
+
+	private void Awake()
+	{
+		itemStorage = GetComponent<ItemStorage>();
+	}
+
+	public void OnSpawnServer(SpawnInfo info)
+	{
+		if (implantObject != null)
+		{
+			Inventory.ServerSpawnPrefab(implantObject, itemStorage.GetIndexedItemSlot(0));
+		}
+	}
+
+	public bool WillInteract(HandApply interaction, NetworkSide side)
+	{
+		if (!DefaultWillInteract.Default(interaction, side)) return false;
+		//can only be applied to LHB
+		if (!Validations.HasComponent<LivingHealthMasterBase>(interaction.TargetObject)) return false;
+
+		if (interaction.Intent != Intent.Help || primed == false) return false;
+
+		if(itemStorage.GetIndexedItemSlot(0).IsEmpty)
+		{
+			Chat.AddExamineMsgFromServer(interaction.Performer, "No implant present in implanter!");
+			return false;
+		}
+
+		return true;
+	}
+
+	public void ServerPerformInteraction(HandApply interaction)
+	{
+		var LHB = interaction.TargetObject.GetComponent<LivingHealthMasterBase>();
```

Comment:
even if it is an acronym, respect the pascalCase rule for private variables. I would personally just call it ``lhb`` or ``health``.